vnext/ReactUWP/EndPoints/dll/react-native-uwp.arm.def

```diff
@@ -1,49 +1,50 @@
+;READFIRST
+;Please keep the ABI surface in sync across all platforms. If you need to add a
+;new EXPORT, add it to the other def files too. The EXPORTs are sorted to aid
+;in comparisons.
 EXPORTS
+??$str_to_floating@N@detail@folly@@YA?AV?$Expected@NW4ConversionCode@folly@@@1@PAV?$Range@PBD@1@@Z
```

Comment:
We should not maintain them manually. Can we do it from script or codegen? will we still need it after we switch to cppwinrt implementation?

Reply:
I hope we still won't need them long term. At the very least, I hope there will only be a very few that will be much more tightly controlled.